### PR TITLE
feat: robusta via knuu

### PR DESCRIPTION
**Pull request by smuu**
## Overview

Based on: https://github.com/celestiaorg/celestia-app/tree/main/test/e2e

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

Closes https://github.com/celestiaorg/devops/issues/364, closes https://github.com/celestiaorg/devops/issues/363,

### CODE REVIEW

File: robusta/txsim.go [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+	//
+	//err = n.Instance.AddFile(genesisFilePath, filepath.Join(remoteRootDir, "config", "genesis.json"), "10001:10001")
+	//if err != nil {
+	//	return fmt.Errorf("adding genesis file: %w", err)
+	//}
+	//
+	//err = n.Instance.AddFile(appConfigFilePath, filepath.Join(remoteRootDir, "config", "app.toml"), "10001:10001")
+	//if err != nil {
+	//	return fmt.Errorf("adding app config file: %w", err)
+	//}
```
Comment (Bidon15) [inline robusta/txsim.go:202]: why is this commented out?

File: robusta/txsim.go [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+	if err != nil {
+		return fmt.Errorf("adding priv_validator_state file: %w", err)
+	}
+
+	err = ts.Instance.AddFile(nodeKeyFilePath, filepath.Join(remoteRootDir, "config", "node_key.json"), "10001:10001")
+	if err != nil {
+		return fmt.Errorf("adding node_key file: %w", err)
+	}
+
+	//err = n.Instance.AddFile(addrBookFile, filepath.Join(remoteRootDir, "config", "addrbook.json"), "10001:10001")
```
Comment (Bidon15) [inline robusta/txsim.go:219]: same for this

**Review by Bidon15 — COMMENTED** [review]
Fantastic!

File: robusta/txsim.go (hunk 1 above)
Comment (smuu) [reply robusta/txsim.go:202]: it's a copy&paste leftover, thanks for mentioning it

**Comment by smuu** [comment]
GH action fixed

**Review by Bidon15 — APPROVED** [review]
🪨

**Pull request merged**